frontend/theta-project-tracker/src/app/components/record-one-week.component.ts

```diff
@@ -1,198 +1,147 @@
-        </ng-container>
-      </form>
-    </div>
-    <ng-container matColumnDef="actions">
-      <th mat-header-cell *matHeaderCellDef></th>
-      <td mat-cell *matCellDef="let project">
-        <button mat-icon-button>
-          <mat-icon>edit</mat-icon>
-        </button>
+      <mat-form-field appearance="outline" formGroupName="normalHours">
+      <input type="number" matInput formControlName="{{ day.name }}">
+    </mat-form-field>
+    <mat-form-field appearance="outline" color="warn" formGroupName="overTime">
+      <input type="number" matInput placeholder="OT" formControlName="{{ day.name }}">
+    </mat-form-field>
+    </form>
+  </mat-grid-tile>
+  <mat-grid-tile
+  class="tile"
+[colspan]="2"
+[rowspan]="1">
+<button mat-icon-button *ngIf="isEditVisible">
+<mat-icon (click)="editRecords()">edit</mat-icon>
+</button>
         <button mat-icon-button>
-          <mat-icon>clear</mat-icon>
+          <mat-icon (click)="onDeleteProject()">clear</mat-icon>
         </button>
-        <button mat-icon-button (click)="onSaveRecords(project)">
+        <button mat-icon-button (click)="getDailyHours()" *ngIf="!isEditVisible">
           <mat-icon>save</mat-icon>
         </button>
-      </td>
-      <td mat-footer-cell *matFooterCellDef></td>
-    </ng-container>
-    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-    <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
-    <tr mat-footer-row *matFooterRowDef="displayedColumns"></tr>
-  </table>
-  <button mat-raised-button color="warn" (click)="onSaveTimeSheet()">Save</button>
+</mat-grid-tile>
+</mat-grid-list>
 </div>
   `,
   styles: [`
   mat-form-field {
-    width: 60px;
+    width: 50px;
   }
   input::-webkit-outer-spin-button,
   input::-webkit-inner-spin-button {
   -webkit-appearance: none;
   margin: 0;
 }
-   table {
-    height: 400px;
+   .wrapper {
     margin: auto;
-    width: 80%;
+    width: 75%;
     overflow: auto;
-    margin-top: 150px
-}
-td {
-  text-align: left;
-  vertical-align: middle;
-}
-.secondary {
-  width: 120px;
-}
+  }
   `]
 })
 export class RecordOneWeekComponent implements OnInit {
-
-  constructor(private authService: AuthService) { }
-  loggedInUser: User;
-  dataSource: ProjectAssigned[];
-  first: ProjectAssigned;
+  @Input() project: ProjectAssigned;
+  days = [
+    { cols: 3, rows: 1, name: 'monday' },
+    { cols: 3, rows: 1, name: 'tuesday' },
+    { cols: 3, rows: 1, name: 'wednesday' },
+    { cols: 3, rows: 1, name: 'thursday' },
+    { cols: 3, rows: 1, name: 'friday' },
+    { cols: 3, rows: 1, name: 'saturday' },
+    { cols: 3, rows: 1, name: 'sunday' },
+  ];
+  @Output() projectEmitter: EventEmitter<Object> = new EventEmitter<Object>();
+  @Output() projectToDelete: EventEmitter<string> = new EventEmitter<string>();
+  @Output() projectToEdit: EventEmitter<string> = new EventEmitter<string>();
+  @Output() cicaEmitter: EventEmitter<number> = new EventEmitter<number>();
```

Comment:
What is this used for? :)

Reply:
hát igen, sajnálom, ez a része nem ment, úgyhogy majd lennének kérdéseim ügyelet alatt:D elég desperate lettem a végére, túl sok időt elbénáztam vele